action.go

```diff
@@ -295,21 +295,27 @@ func (c *cmdAction) ClusterCanRetry() bool {
 // set Nil to true. If not the return value will be unmarshaled into Rcv
```

Comment:
Can you update the comment to read as:

> MaybeNil is a type which wraps a response receiver. If the response being received is a nil RESP type (either bulk string or array), it will set Nil to true. If not the return value will be unmarshaled into Rcv normally. If the response being received is an empty array then the `EmptyArray` field will be set and Rcv unmarshaled into normally.